staging/src/k8s.io/legacy-cloud-providers/azure/azure_cache.go

```diff
@@ -122,12 +126,16 @@ func (t *timedCache) Get(key string, crt cacheReadType) (interface{}, error) {
 
 	// entry exists
 	if entry.data != nil {
+		// force refresh the cache to get the current accurate data
+		if crt == cacheReadTypeForceRefresh {
+			klog.V(5).Infof("Forcing refreshing the cache for %s", key)
```

Comment:
I think you can move this to L140 without an if-statement. print out crt in log if you worry we will add new type in future.

Reply:
Updated!